hie-compat/src-ghc901/Compat/HieBin.hs

```diff
@@ -0,0 +1,371 @@
+{-
```

Comment:
Does this need to be duplicated? I think it should be possible to just reexport `GHC.Iface.Ext.Binary`

Reply:
Not sure. I just copied whatever was in the ghc-810 versions and updated it to the ghc-901 version.